phpswitch.sh

```diff
@@ -68,9 +74,14 @@ then
 			echo "Switching your apache conf"
 			for j in ${php_installed_array[@]}
 			do
-				sudo sed -i.bak "s/^LoadModule[ \t]php5_module[ \t]$php_opt_path$j$apache_php_lib_path/\#LoadModule php5_module $php_opt_path$j$apache_php_lib_path/g" $apache_conf_path
```

Comment:
@mirfilip this part is required to comment out all existing PHP modules in apache conf not just the one being selected.